lib/ansible/modules/windows/win_dsc.ps1

```diff
@@ -374,7 +374,9 @@ if ($resource.Module) {
 # engine updates the metadata propery. We don't care about any errors here
 try {
     Invoke-DscResource -Method Get -Property @{Fake="Fake"} @dsc_args > $null
-} catch {}
+} catch {
+    Out-Null;
```

Comment:
is trailing ; on the Out-Null needed?

Reply:
No, in powershell you can leave out ; , but im used to writing with and i guess that one slipped